PF1/Assignment1/OctalConverter.java

```diff
@@ -0,0 +1,56 @@
+package assignment1;
+// 
+public class OctalConverter {
+	
+		int binaryToOctal(int binum)
+		{
+			int temp=0;
+			try
+			{int no=binum;
+				
+				while(no>0){
+				if(no%10!=1 | no%10!=0)
+				{
+					throw new Exception();
+				}
+				no=no/10;
+				}
+			}
+			catch(Exception e)
+			{
+
+	System.out.println("wrong Input");
+		}
+			
+			int dnum=0;
```

Comment:
This logic is working fine, but as discussed in review meeting try to implement with alternate logic also.